eng/ci/templates/official/jobs/build-test.yml

```diff
@@ -250,6 +251,25 @@ jobs:
     env:
       IntegrationBuildNumber: $(INTEGRATIONBUILDNUMBER)
     displayName: 'Move artifacts'
+
+  - ${{ if eq(variables['Build.Reason'], 'Schedule') }}:
+    - task: PowerShell@2
+      displayName: 'Generate nightly build package version for feed'
+      inputs:
+        targetType: filePath
+        filePath: '$(Build.SourcesDirectory)/generateNightlyBuildPackageVersion.ps1'
+
+  - ${{ if eq(variables['Build.Reason'], 'Schedule') }}:
```

Comment:
Given these two tasks are after each other, i dont think you need to repeat `${{ if eq(variables['Build.Reason'], 'Schedule') }}:`

They can be both indented under the first instance of this if condition